docs/source/install.rst

```diff
@@ -23,8 +23,17 @@ single tool to install Python and non-Python dependencies, or if you're already
 using `Anaconda <http://continuum.io/downloads>`_ as your Python distribution,
 you can skip to the :ref:`Installing with Conda <conda>` section.
 
+Since building Zipline's C extensions requires numpy's CPython C API as well as
+`Cython <http://cython.org/>`_, pip install these before pip installing zipline
+itself:
+
+.. code-block:: bash
+
+   $ pip install numpy Cython
```

Comment:
This will give people the latest version of numpy.  Do we support that?

Reply:
AFAIK we do. If we don't, then we should update setup.py's `REQ_UPPER_BOUNDS`.